Generated code does not compile

The Docker build is currently broken at 0d21aed due to incorrect generated code

```
$ docker build --no-cache .
Sending build context to Docker daemon 501.9 MB
Step 1/17 : ARG SPARK_IMAGE=gcr.io/spark-operator/spark:v2.4.0
Step 2/17 : FROM golang:1.12.3-alpine as builder
1.12.3-alpine: Pulling from library/golang
bdf0201b3a05: Pull complete 
38f114998adb: Pull complete 
21134b1a9e68: Pull complete 
06fc59056607: Pull complete 
f53f6727338a: Pull complete 
Digest: sha256:1e05444cc4070a7eb4acdb47077dcac8d21489455a0a1ffb4de52cfef8d59c00
Status: Downloaded newer image for golang:1.12.3-alpine
 ---> 821acdc20eb8
Step 3/17 : ARG DEP_VERSION="0.5.1"
 ---> Running in 7a89ed9832ef
Removing intermediate container 7a89ed9832ef
 ---> ba854a76bbe2
Step 4/17 : RUN apk add --no-cache bash git
 ---> Running in 852f66a4e009
fetch http://dl-cdn.alpinelinux.org/alpine/v3.9/main/x86_64/APKINDEX.tar.gz
fetch http://dl-cdn.alpinelinux.org/alpine/v3.9/community/x86_64/APKINDEX.tar.gz
(1/11) Installing ncurses-terminfo-base (6.1_p20190105-r0)
(2/11) Installing ncurses-terminfo (6.1_p20190105-r0)
(3/11) Installing ncurses-libs (6.1_p20190105-r0)
(4/11) Installing readline (7.0.003-r1)
(5/11) Installing bash (4.4.19-r1)
Executing bash-4.4.19-r1.post-install
(6/11) Installing nghttp2-libs (1.35.1-r0)
(7/11) Installing libssh2 (1.8.2-r0)
(8/11) Installing libcurl (7.64.0-r1)
(9/11) Installing expat (2.2.6-r0)
(10/11) Installing pcre2 (10.32-r1)
(11/11) Installing git (2.20.1-r0)
Executing busybox-1.29.3-r10.trigger
OK: 29 MiB in 26 packages
Removing intermediate container 852f66a4e009
 ---> 54a3b71f4a63
Step 5/17 : ADD https://github.com/golang/dep/releases/download/v${DEP_VERSION}/dep-linux-amd64 /usr/bin/dep
Downloading [==================================================>] 11.05 MB/11.05 MB
 ---> f1655eb86f55
Step 6/17 : RUN chmod +x /usr/bin/dep
 ---> Running in bee32dd2ecdd
Removing intermediate container bee32dd2ecdd
 ---> 5e3c78565899
Step 7/17 : WORKDIR ${GOPATH}/src/github.com/GoogleCloudPlatform/spark-on-k8s-operator
Removing intermediate container 1f25cdde26df
 ---> 74a9cf481ebc
Step 8/17 : COPY Gopkg.toml Gopkg.lock ./
 ---> 10b2aa210442
Step 9/17 : RUN dep ensure -vendor-only
 ---> Running in 2642e59a2194
Removing intermediate container 2642e59a2194
 ---> 4d55b3477ab0
Step 10/17 : COPY . ./
 ---> 7317868d05f5
Step 11/17 : RUN go generate && CGO_ENABLED=0 GOOS=linux go build -o /usr/bin/spark-operator
 ---> Running in 5f4a88eea620
Generating deepcopy funcs
Generating clientset for sparkoperator.k8s.io:v1alpha1,v1beta1 at github.com/GoogleCloudPlatform/spark-on-k8s-operator/pkg/client/clientset
Generating listers for sparkoperator.k8s.io:v1alpha1,v1beta1 at github.com/GoogleCloudPlatform/spark-on-k8s-operator/pkg/client/listers
Generating informers for sparkoperator.k8s.io:v1alpha1,v1beta1 at github.com/GoogleCloudPlatform/spark-on-k8s-operator/pkg/client/informers
# github.com/GoogleCloudPlatform/spark-on-k8s-operator/pkg/client/clientset/versioned/typed/sparkoperator.k8s.io/v1beta1
pkg/client/clientset/versioned/typed/sparkoperator.k8s.io/v1beta1/sparkoperator.k8s.io_client.go:80:45: scheme.Codecs.WithoutConversion undefined (type serializer.CodecFactory has no field or method WithoutConversion)
# github.com/GoogleCloudPlatform/spark-on-k8s-operator/pkg/client/clientset/versioned/typed/sparkoperator.k8s.io/v1alpha1
pkg/client/clientset/versioned/typed/sparkoperator.k8s.io/v1alpha1/sparkoperator.k8s.io_client.go:80:45: scheme.Codecs.WithoutConversion undefined (type serializer.CodecFactory has no field or method WithoutConversion)
The command '/bin/sh -c go generate && CGO_ENABLED=0 GOOS=linux go build -o /usr/bin/spark-operator' returned a non-zero code: 2
```

--- a/Gopkg.lock
+++ b/Gopkg.lock
@@ -1066,8 +1066,7 @@
   version = "kubernetes-1.13.5"
 
 [[projects]]
-  branch = "master"
-  digest = "1:bcdda675481aa9cad7586932de7077c107a76d2157872e4ae37d7a41ebd9bfe2"
+  digest = "1:d809e6c8dfa3448ae10f5624eff4ed1ebdc906755e7cea294c44e8b7ac0b077a"
   name = "k8s.io/code-generator"
   packages = [
     "cmd/client-gen",
@@ -1082,11 +1081,11 @@
     "cmd/deepcopy-gen/args",
     "cmd/defaulter-gen",
     "cmd/defaulter-gen/args",
-    "pkg/namer",
     "pkg/util",
   ]
   pruneopts = ""
-  revision = "826a68e0d1203c132e5c01e06e271db403b08701"
+  revision = "c2090bec4d9b1fb25de3812f868accc2bc9ecbae"
+  version = "kubernetes-1.13.5"
 
 [[projects]]
   branch = "master"

--- a/Gopkg.toml
+++ b/Gopkg.toml
@@ -98,8 +98,8 @@ required = [
   version = "1.1"
 
 [[constraint]]
-  branch = "master"
   name = "k8s.io/code-generator"
+  version = "kubernetes-1.13.5"
 
 [[constraint]]
   name = "github.com/evanphx/json-patch"

